edisgo/flex_opt/reinforce_grid.py

```diff
@@ -746,6 +776,7 @@ def reinforce():
             "reinforcement."
         )
         selected_timesteps = non_converging_timesteps
+        troubleshooting_mode = False
```

Comment:
Here I turned off troubleshooting mode, as in case power flow still not converges for all time steps, we want to go straight to the iterative grid reinforcement instead of running the power flow with reduced time series until it converges in get_most_critical_time_steps(). However, now that I think of it a bit more, the troubleshooting mode probably never makes sense for the grid reinforcement because it is only used to determine the most critical time steps, but in power flows afterwards in reinforce_grid() it would still results in convergence issues. Or can you think of a case where it might be helpful to have that parameter in the reinforcement?

Reply:
From the top of my head, I cannot think of any.

Reply:
I set the default to False, as it makes more sense. The user can still set it to True if he or she wants to.